dnf-behave-tests/dnf/upgrade.feature

```diff
@@ -187,17 +188,20 @@ Scenario: Print information about skipped packages
     And Transaction is following
         | Action        | Package                                   |
         | upgrade       | SuperRipper-0:1.2-1.x86_64                |
-        | broken        | SuperRipper-0:1.3-1.x86_64                |
    Then stdout section "Upgrading:" contains "SuperRipper\s+x86_64\s+1.2-1\s+dnf-ci-thirdparty-updates"
```

Comment:
Since we have the step above "Transaction is following" this feels redundant to me.
We already check that `SuperRipper-0:1.2-1.x86_64` gets upgraded.
Is there some reason to keep it? 
It would also unblock the PR.

Reply:
It was present in the original version of the scenario, I supposed it tests that the output is exactly as it is described in the scenario. If not, removing.